INSTRUCTION.md

```diff
@@ -0,0 +1,8 @@
+#### Create cluster
+kind create cluster --config cluster.yml
+
+#### Taint nodes labeled
+kubectl taint nodes -l app=mysql app=mysql:NoSchedule
+
+#### Deploy using helm
+helm install todoapp .infrastructure/helm-chart/todoapp
```

Comment:
Ensure that the path `.infrastructure/helm-chart/todoapp` is correct and accessible from your current working directory. If the path is incorrect, the Helm install command will fail. Consider verifying the path or providing a relative path from the current directory.